test/jdk/java/awt/event/MouseEvent/AWTPanelSmoothWheel.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2009, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Frame;
+import java.awt.Panel;
+
+/*
+ * @test
+ * @bug 6730447
+ * @summary Support for high resolution mouse wheel is still incomplete.
+ *          AWT panel needs to be supported
+ * @requires (os.family == "windows")
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual AWTPanelSmoothWheel
+ */
+
+public class AWTPanelSmoothWheel {
+    private static JTextArea textArea;
+    public static final String INSTRUCTIONS = """
+            This test is relevant for windows platforms and mouses with high-resolution wheel,
+            please just press pass if this is not the case.
+            
```

Comment:
@prrace I wanted to check the relevance of the testing requirement. The original test was created for Windows Vista and for a high-res mouse. I do think the test can be retained to check on newer versions of windows. Any suggestions?

Reply:
A trackpad is usually a high-res mouse wheel, it allows smooth scrolling (as opposed to moving 3 lines on a wheel notch).

There are mice which also support smooth scrolling, I've got one.